src/code42cli/click_ext/options.py

```diff
@@ -0,0 +1,36 @@
+import click
```

Comment:
has anything changed here?

Reply:
Does not look like this - this was in `options.py` before, if that helps.